cmd/m3db-operator/main.go

```diff
@@ -53,7 +53,7 @@ import (
 
 const (
 	// Informers will resync on this interval
-	_informerSyncDuration = 30 * time.Second
+	_informerSyncDuration = 5 * time.Minute
```

Comment:
Why the increase?

Reply:
It made the logs super spammy, and we do a good job of reacting to events so I didn't feel it was necessary to have such a frequent resync. I browsed some other operators code and it seems like they do 5m-30m